openslides/motions/models.py

```diff
@@ -729,14 +730,14 @@ class State(RESTModelMixin, models.Model):
     action_word = models.CharField(max_length=255)
     """An alternative string to be used for a button to switch to this state."""
 
-    workflow = models.ForeignKey('Workflow')
+    workflow = models.ForeignKey('Workflow', related_name='states')
     """A many-to-one relation to a workflow."""
 
     next_states = models.ManyToManyField('self', symmetrical=False)
     """A many-to-many relation to all states, that can be choosen from this state."""
 
-    icon = models.CharField(max_length=255)
-    """A string representing the url to the icon-image."""
+    css_class = models.CharField(max_length=255)
```

Comment:
I added `default='primary'` because the blue labels are the default color. `default` is set for undecided end states, rejected for accept, green for accepted.